### PR TITLE
CY-227-Update-Tooltip-Component

### PR DESCRIPTION
**Pull request by nicolasshiken**

### CODE REVIEW

File: src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -2,11 +2,14 @@ import React, { ReactNode } from "react";
...
+
 export interface ITooltip {
   content: ReactNode | (() => ReactNode);
   supressArrow?: boolean | undefined;
   placement?: TPlacement | undefined;
-  color?: string | undefined;
+  color?: TTooltipColor;
 }
 
 export const Tooltip: React.FC<ITooltip> = ({ ...props }) => {
```
Comment (diegomart2000) [inline src/components/tooltip/Tooltip.tsx]: Use destructuring and named imports

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

File: src/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
   content: ReactNode | (() => ReactNode);
-  supressArrow?: boolean | undefined;
   placement?: TPlacement | undefined;
-  color?: string | undefined;
+  color?: TTooltipColor;
 }
 
-export const Tooltip: React.FC<ITooltip> = ({ ...props }) => {
+export const Tooltip: FC<ITooltip> = ({ content, placement, color, children }) => {
   // console.log(`Tooltip props: `, props);
```
Comment (diegomart2000) [inline src/components/tooltip/Tooltip.tsx:15]: @nicolasshiken please remove this line

**Review by diegomart2000 — CHANGES_REQUESTED** [review]
Looking good. Just a super minor comment

File: src/components/tooltip/styles/STooltip.tsx [hunk 3]
```diff
@@ -3,22 +3,22 @@ import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../consta
...
-  font-size: 12px;
-  line-height: 1.67;
-  font-weight: 600;
-  border-radius: 6px;
-  background-color: ${(props) => (props.color ? props.color : "var(--neutral-600)")};
-  color: var(--ui-01);
+  border-radius: 4px;
+  background-color: ${(props) => props.backgroundColor};
+  color: ${(props) => props.color};
   animation: ${(props) => (props.closing ? "close" : "open")} 100ms ease-in-out;
```
Comment (diegomart2000) [inline src/components/tooltip/styles/STooltip.tsx:18]: @nicolasshiken Testing the black mode you will notice that there is a timing difference between the body of the tooltip and its tip. Please make them work in synchronization, to appear and disappear at the same time.

File: src/components/tooltip/Tooltip.stories.jsx [hunk 4]
```diff
@@ -1,156 +1,78 @@
...
+      <h1 style={{ fontSize: 14, fontWeight: "bold", textAlign: "center" }}>A friendly reminder</h1>
+      <p
+        style={{
+          fontSize: 12,
+          lineHeight: "16px",
+          textAlign: "center",
+          fontWeight: "lighter",
+        }}
+      >
+        You can save your search view and come back to it later just in one click
```
Comment (diegomart2000) [inline src/components/tooltip/Tooltip.stories.jsx:52]: @nicolasshiken Please add a story with an check icon like in designs for testing.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

File: src/components/tooltip/styles/STooltip.tsx (hunk 3 above)
Comment (nicolasshiken) [reply src/components/tooltip/styles/STooltip.tsx:18]: Yes I noticed that! But I think that is related to the Popover component, so maybe I can fix that on another PR?

Comment (diegomart2000) [reply src/components/tooltip/styles/STooltip.tsx:18]: Yeah, that works too, will create a new ticket.

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**